src/main/java/unicash/ui/StyleSheet.java

```diff
@@ -20,4 +20,89 @@ public class StyleSheet {
     // Custom text formatting
     public static final String TRANSACTION_ID_SEPARATOR = ". ";
 
+    // Color utilities
+    public static final String TEXT_BACKGROUND_COLOR = "-fx-background-color: %s";
+    public static final int MAX_COLOR_VALUE = 255;
+    public static final int HEXADECIMAL_RADIX = 16;
+
+    // Color variables
+    public static final int HEX_COLOR_START_INDEX = 0;
+    public static final int HEX_COLOR_END_INDEX = 6;
+    public static final int BRIGHTNESS_THRESHOLD = 135;
+    public static final int BRIGHTNESS_OFFSET = 64;
+    public static final int RED_OFFSET = 30;
+    public static final int GREEN_OFFSET = 20;
+    public static final int BLUE_OFFSET = 00;
+    public static final boolean IS_YELLOW_SKEW = true;
+
+    /**
+     * Returns a 6-digit hexadecimal number based on an Object's unique
+     * hash code, to be used as a color indicator. If the hex color value
+     * generated from the hashcode accords poor {@code Label} visibility,
+     * the value is offset such that is always of a certain brightness value.
+     */
+    public static String getBrightCategoryColorFromHash(Object obj) {
+
+        // Absolute value of hash code taken to guard against negative values
+        String hexString = Integer.toHexString(Math.abs(obj.hashCode()));
+
+        // Padding for hash codes that are less than 6 digits long
+        while (hexString.length() < 6) {
+            hexString = "0" + hexString;
+        }
```

Comment:
Is it possible to not hardcode this? Since this is assuming the hexString is always at least 5 charcaters long right. Like some padding function by String